app/controllers/steps/capital/add_saving_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Steps
+  module Capital
+    class AddSavingController < Steps::CapitalStepController
```

Comment:
@hiboabd What do you think of using "SavingsController" instead of "AddSavingsController" ?